build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -387,6 +392,11 @@ antreaProxy:
   # Note that this option is experimental. If kube-proxy is removed, option kubeAPIServerOverride must be used to access
   # apiserver directly.
   proxyAll: {{ .proxyAll }}
+  # hostNetworkMode tells antrea-agent how to enforce netfilter rules required by `proxyAll` in the Node's host network.
+  # The default value is "iptables". If "nftables" is specified, the HostNetworkNFTables feature gate must be enabled;
+  # otherwise, this option has no effect. If the above condition is met but nftables is not supported or unavailable on
+  # the Node, antrea-agent will fail to start.
+  hostNetworkMode: {{ .hostNetworkMode | quote }}
```

Comment:
IMO, this should be a top-level config, and not nested under `antreaProxy`: we will use the same config for all features

Reply:
Done